src/kata6.joinNames.js

```diff
@@ -1,3 +1,12 @@
-const joinNames = (namesObj) => {};
+const joinNames = (namesObj) => {
+   let simpNames = []
```

Comment:
And the simpNames array is never reassigned, therefor it should be const instead of let.
I am guessing the confusion might be around modifying the array. Modifying and reassigning are two different things. When you declare an object or an array with const you are still able to modify the values of the variable.